### DISCUSSION TITLE
need help to resolve problem

### DISCUSSION THREAD

**Discussion by Raza-Webiators** [breezefront/community#161] [1. Help]
there is only show header and how i will set its layout

**Comment by vovayatsyuk** [comment]
Sorry, but I don't understand your question. Could you be more specific?

**Reply by Raza-Webiators** [reply]
i am installing breeeze theme but they only considerd header they couldn't show content and footer and with backend how i configured the layout

**Reply by vovayatsyuk** [reply]
Did you run the installer? What was the output in the terminal?
